corehq/apps/hqwebapp/static/hqwebapp/less/_hq/optimize.less

```diff
@@ -0,0 +1,39 @@
+/* styles for Google Optimize A/B tests */
```

Comment:
My read of this is that right now, the control elements will always display and this PR doesn't contain code to actually turn on the AB test, is that right?

Reply:
correct. the A/B test is turned on with Google Optimize

Reply:
by adding a class to the wrapper that is 'show-a' or 'show-b'